ledger/catchupaccessor.go

```diff
@@ -493,10 +515,14 @@ func (c *CatchpointCatchupAccessorImpl) processStagingBalances(ctx context.Conte
 	if errHashes != nil {
 		return errHashes
 	}
+	if errKVs != nil {
+		return errKVs
+	}
```

Comment:
I think this file is missing an update in `finishBalances` - the code around `RunMigrations`
we shortcut when implemented the online tables, so catchpoint has v6 tables, and them migrates DB to v7, and the migration converts accountbase into online accounts.
For v8 schema and boxes in catchpoint this does not work anymore. I'll see what can be done and commit into this PR to fix this.

Reply:
It appears to work fine, my bad.
The only issue we cannot remove v6->v7 migration from catchpoint apply code b/c it also removes state proof data from offline accounts.